support/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge.md

```diff
@@ -21,9 +21,10 @@ Windows desktop applications that have a dependency on the C++ Runtime libraries
 
 Microsoft provides C++ Runtime framework packages to allow applications to reference the C++ runtime from desktop applications distributed through the Windows Store. These packages are distributed and updated through the Windows Store and are handled similarly to C++ UWP framework packages.
 
-For development purposes, the current version (v14.0) of both debug and retail appx packages are included with Visual Studio 2019 when you choose the **Universal Windows Platform Development** workload with the optional **C++ (v142) Universal Windows Tools** component. The packages can be found under `%ProgramFiles(x86)%\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs.Desktop\14.0`.
+For development purposes, the current version (v14.0) of both debug and retail appx packages are included with Visual Studio 2022 when you choose the **Universal Windows Platform Development** workload with the optional **C++ (v143) Universal Windows Tools** component. The packages can be found under `%ProgramFiles(x86)%\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs.Desktop\14.0`.
 
-In some scenarios such as [Windows Sandbox](/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview) or where applications run on offline machines, developers may find it easier to download the packages corresponding to their deployment architectures from one of the links below and manually install them using the `Add-AppxPackage` PowerShell cmdlet:
+**Note:**
```

Comment:
For note, we use a specific syntax, which I've edited your PR to have.
`

Reply:
Thank you, @TylerMSFT ! Fixed.